adafruit_esp32spi/adafruit_esp32spi_wsgiserver.py

```diff
@@ -216,3 +217,11 @@ def _get_environ(self, client):
             env[key] = value
 
         return env
+
+    def check_remote_ip(self):
```

Comment:
I would move this to the `Socket` class as a `@property`, like we did for `socknum`.(https://github.com/adafruit/Adafruit_CircuitPython_ESP32SPI/blob/master/adafruit_esp32spi/adafruit_esp32spi_socket.py#L183-L186)
We could name the property 'remote_ip`. You could add in a second property for `remote_port` as well.

then every socket instance could get the remote_ip by doing `sock_instance.remote_ip`, and the port by doing `sock_instance.remote_port`